iron-dropdown.html

```diff
@@ -192,10 +197,7 @@
          * Overridden from `IronOverlayBehavior`.
          */
         _renderOpened: function() {
-          if (!this.noAnimations && this.animationConfig && this.animationConfig.open) {
-            if (this.withBackdrop) {
-              this.backdropElement.open();
-            }
+          if (!this.noAnimations && this.animationConfig.open) {
```

Comment:
Because it gets set in `_updateAnimationConfig`, which is invoked on `_openedChanged`.